src/generators/generateActionStubs.ts

```diff
@@ -51,15 +51,24 @@ export class GenerateActionStubs {
     if (!response?.assistants) {
       return;
     }
-
-    for (const sig of response?.assistants) {
-      let allParamsSorted = this.getAllParamsSorted(sig, stubs);
-      pythonContent += `@assistant('${sig.id}')\n`;
-      pythonContent += this.buildAssistantParmDec(sig, allParamsSorted);
-      pythonContent += `async def assistant_${snakeCase(sig.name)}(\n`;
-      pythonContent += this.buildAssistantParams(sig, allParamsSorted);
-      pythonContent += `) -> None:\n`;
-      pythonContent += `    pass\n\n\n`;
+    const labs = await client.queryLabs();
+    if (!labs) {
+      return;
+    }
+    for (const lab of labs.labs) {
+      pythonContent += `class ${lab.name.charAt(0).toUpperCase() + camelCase(lab.name).slice(1)}Assistants:\n`;
+      for (const sig of response.assistants) {
+        if (lab.id === sig.constraint.labId) {
+          let allParamsSorted = this.getAllParamsSorted(sig, stubs);
+          pythonContent += `\t@staticmethod\n`;
```

Comment:
Does putting tab characters in the generated code mess with the linter?

Reply:
I turn off flake8 in the generated file, so i mostly roughly make it look 'good' and i think it generally follows the guidelines but i know of a few violations.  If you save the file pep8 will shift some things around, and if you regenerate it again, it will shift them back.  

I havent tried to make it perfectly stable wrt whitespace/tabs etc.. as editing and saving the generated file is generally a bad idea anyways.